src/docfx/lib/log/ErrorLog.cs

```diff
@@ -32,29 +33,35 @@ internal sealed class ErrorLog : IDisposable
 
         public int SuggestionCount => _suggestionCount;
 
-        public ErrorLog(string docset = ".", bool legacy = false)
+        public ErrorLog(
+            string command, string docsetPath, string outputPath, Func<Config> config, bool legacy = false)
         {
-            _docsetPath = docset;
+            _command = command;
+            _config = config;
             _legacy = legacy;
-            _config = new Config();
+            _stopwatch = Stopwatch.StartNew();
```

Comment:
why is `stopwatch` related to `ErrorLog`?

Reply:
updated